### ISSUE TITLE
Long text in prompt breaks Flux dev

### ISSUE THREAD

**Issue by Volnovik** [lllyasviel/stable-diffusion-webui-forge#1830]
I tried creating an image and just could not get text right.
This made me want to compare text capabilities of Flux dev model, so I got to ideaogram and took first nice looking prompt with long text:
[https://ideogram.ai/g/q5QR1-OGRCOrGj9e7nFT-g/0](https://ideogram.ai/g/q5QR1-OGRCOrGj9e7nFT-g/0)
Prompt is:
`A stunning and vibrant 3D illustration of a wall art piece, featuring the word 'FOCUS' in eye-catching, colorful, and uniquely designed crocheted letters. Each letter is in a different vibrant colour. The bold white text below the main word reads, "ON YOUR DREAMS AND THEY WILL BECOME YOUR REALITY." The artwork is adorned with an array of decorative elements, including flowers, iridescent bubbles, and butterflies, all set against a gradient background that transitions from a light pinkish hue at the top to a deeper purple at the bottom. The scene is set against a plain gray floor and a white ceiling with recessed lights, creating a captivating and inspiring visual experience., photo, illustration, 3d render, typography`
Pasted it in my setup with full flux.dev and t5xxlfp16. Result had me question stuff:
![00327-2345183254](https://github.com/user-attachments/assets/49c71ca8-0a88-4d60-a7a2-58d068cb7fd3)

I could no believe Flux is THAT bad with text so headed to online generaion service to check pro and it did well. Made me want to check dev model there and it also performed well. Space on huggingface - also amazing image. Turned back to tinkering with samplers, schedule types etc - nothing worked. Then I reduced text in prompt to:
`A stunning and vibrant 3D illustration of a wall art piece, featuring the word 'FOCUS' in eye-catching, colorful, and uniquely designed crocheted letters. Each letter is in a different vibrant colour. The bold white text below the main word reads, "**ON YOUR DREAMS**" The artwork is adorned with an array of decorative elements, including flowers, iridescent bubbles, and butterflies, all set against a gradient background that transitions from a light pinkish hue at the top to a deeper purple at the bottom. The scene is set against a plain gray floor and a white ceiling with recessed lights, creating a captivating and inspiring visual experience., photo, illustration, 3d render, typography`
And it worked just fine:
![00331-1825267790](https://github.com/user-attachments/assets/d8f18f8b-4882-4199-bb00-f3e43b14d048)
No need to be an expert to see difference.

Installed comfy with swarmui to check if my pc has some issues and no, in comfy everything is fine:
![0027-A stunning and vibrant 3D illustration o-flux_dev-690774450](https://github.com/user-attachments/assets/5c59209b-56ed-45a7-a6d4-bed1657a86fb)
Moreover, previous prompt that started this all and gave good image with wrong text generated proper text on second try:
![0030-A high-quality image of a muscular bald-flux_dev-1041764079](https://github.com/user-attachments/assets/1905b688-2ab1-4d19-9676-bd600ccffb37)

I tried reloading Forge, tried Q8 gguf etc, no good. My flux related parameters are:
![01-2024-09-16 004656](https://github.com/user-attachments/assets/7eb60555-dcc7-4594-818c-92a489ba7a52)
Forge is used inside StabilityMatrix, same for Swarmui install. GPU is 4090.
Updated last on 15.09.24
No Loras or extensions enabled.
Also noticed that with ideogram prompt generation time was longer. No errors in console, but in case of prompt from ideogram it was significantly slower that usual, 1.66s/it vs 1.3 it/s on other prompts.

So to summarize:

- Long text works worse than in other UIs
- Certain prompts with long text completely breaks the image and slows generation.
- Reducing text length in prompt "fixed" the issue.
- Clearly some bug under the hood.

**Comment by Juqowel** [comment]
No, it's not about the prompt length. BUG in the part that is written in capital letters.
Same first prompt, but lowercase "on your dreams and they will become your reality":

![00003-1780770663](https://github.com/user-attachments/assets/ffe09961-0ae9-483d-9143-a2f9a400cf7f)

**Comment by Volnovik** [comment]
Not prompt length but length of text inside prompt. Works fine with upper case and less text. But [Juqowel](https://github.com/Juqowel) is also right, if you make all text lower case I seem to get better text (anecdotal, 9 generations, no gibberish, but missing words sometimes). Also, if I replace second text line with gibberish in upper case - it generates the image and is not breaking it:
"WEB WT ER WEQWERTQWERTR ERTWERT."
![00020-4115921302](https://github.com/user-attachments/assets/6da456c0-0a23-46c8-b4e3-11670e5a4a2a)

**Comment by Juqowel** [comment]
Okay, I forgot about that.. It cause by "AND" keyword.
It should be: "ON YOUR DREAMS \A\N\D THEY WILL BECOME YOUR REALITY."

![00007-1780770663](https://github.com/user-attachments/assets/c488c1c2-4dad-40e0-b177-1131640be944)

**Comment by Volnovik** [comment]
Update:
As Demian Dei from discord pointed this is related to 
[composable diffusion](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#composable-diffusion) feature of A1111. 
AND is detected as special keyword and slaughters image. BREAK also does not work as intended and seems to just null everything after.
Guess we need a toggle to disable all that preprocessing for flux.
